cmd/validator/validator.go

```diff
@@ -133,6 +143,43 @@ func getFlags() (validatorConfig, error) {
 		}
 	}
 
+	uncheckedClean := cleanString("unchecked-files")
+	uncheckedPairs := strings.Split(uncheckedClean, ",")
+
+	// Check that checked files are listed in proper format and with valid config types
+	if uncheckedFiles != nil && isFlagSet("unchecked-files") {
+		for _, pair := range uncheckedPairs {
+			if !strings.Contains(pair, ":") {
+				err:= fmt.Errorf("Format error in adding unchecked file: %s, ':' delimeter required to separate file path from type", pair)
+				fmt.Println(err)
+				flag.Usage()
+				return validatorConfig{}, err
+			}
+			uncheckedType := strings.Split(pair, ":")
```

Comment:
```suggestion
			uncheckedType := strings.Split(pair, ":")
     key, val := uncheckedType[0], uncheckedType[1]
```

Or any meaningful name for key & val variables

Then use these variables instead of cryptic `uncheckedType[0]` & `uncheckedType[1]`

Reply:
You can even use 

```go
key, val := strings.Cut(pair, ":")
```